packages/webpack-config/webpack/createClientEnvironment.js

```diff
@@ -5,7 +5,7 @@ module.exports = function createClientEnvironment(mode, publicPath, nativeAppMan
   const environment = mode || 'development';
   const __DEV__ = environment !== 'production';
 
-  const ENV_VAR_REGEX = /^(EXPO_|REACT_NATIVE_)/i;
+  const ENV_VAR_REGEX = /^(EXPO_|REACT_NATIVE_|CI)/i;
```

Comment:
change this to be more restrictive -- if someone has a variable like CI_SECRET_TOKEN we don't want that passed through I think

```suggestion
  const ENV_VAR_REGEX = /^(EXPO_|REACT_NATIVE_|CI$)/i;
```